models/bpreg/utils/BpRegRunner.py

```diff
@@ -0,0 +1,37 @@
+"""
+---------------------------------------------------------
+Author: Suraj Pai, Leonard Nürnberg 
+Email:  bspai@bwh.harvard.edu, lnuernberg@bwh.harvard.edu
+Date:   06.03.2024
+---------------------------------------------------------
+"""
+import os
+from pathlib import Path
+from mhubio.core import Instance, InstanceData, IO, Module
+from bpreg.inference.inference_model import InferenceModel
+import torch
+
+
+class BpRegRunner(Module):
+    """
+    BpRegRunner is a module for running body part regression on CT images.
+    It takes a NIfTI CT image as input and outputs a JSON file with extracted features.
+    """
+    
+    @IO.Instance()
+    @IO.Input('in_data', 'nifti:mod=ct', the='Input nifti ct image file')
+    @IO.Output('utility_json', 'utility.json', "json:type=utility", bundle='model', the='Features extracted from the input image')
+    def task(self, instance: Instance, in_data: InstanceData, utility_json: InstanceData) -> None:
+        """
+        Perform body part regression on the input NIfTI CT image and save the extracted features to a JSON file.
+
+        Args:
```

Comment:
To avoid redundant annotations, please use the `the=` descriptor in the decorators instead.